api/app/main.py

```diff
@@ -145,10 +148,10 @@ async def sync_user(msg: SyncUserRequest, force: bool = False):
         }
 
     submissions = response.json().get('result', [])
+    submissions = list(reversed(submissions))
     for submission in submissions:
-        cf_reference = submission['id']
+        problem_key = f"{submission['problem']['contestId']} + {submission['problem']['index']}"
```

Comment:
what's the + sign here for ?

Reply:
This is supposes to conatenate the contest ID and the index of the problem in it's contest in order to make unique key for the taken problem.